tm2/pkg/sdk/context.go

```diff
@@ -147,31 +147,21 @@ func (c Context) WithEventLogger(em *EventLogger) Context {
 	return c
 }
 
-// WithValue is deprecated, provided for backwards compatibility
```

Comment:
was intended to be deprecated.

Reply:
See-also previous conversation: https://github.com/gnolang/gno/pull/2319#discussion_r1727571706